policy/modules/apps/gnome.te

```diff
@@ -80,6 +80,8 @@ optional_policy(`
 # Conf daemon local Policy
 #
 
+allow gconfd_t self:unix_stream_socket { accept listen };
```

Comment:
Please use `create_stream_socket_perms` instead.

Reply:
I can't foresee a valid reason to add the create socket permissions.

Please see https://github.com/SELinuxProject/refpolicy/pull/639#discussion_r1316142515 for reference about this issue.

Reply:
I have updated the discussion, please see https://github.com/SELinuxProject/refpolicy/pull/639#discussion_r1316346698

Reply:
I think they are inherited socket file descriptors in this case.

Reply:
It might related to dbus...

Reply:
For example, dbus_all_session_domain(), which would prove it does not be granted the "create" socket permission.

What do you say ?

Reply:
I think the socket is created by dbus, which then launches gconfd and the latter inherits the socket file descriptor.

Reply:
I have just modified the dbus policy in order to add the ability to create sockets:

https://github.com/SELinuxProject/refpolicy/pull/667

This is backed by a source code audit...

Similarly I have searched the gconf source code and found no calls to socket() creation function and this reinforces my previous comment. It must be sockets inherited from the dbus daemon process after it executes the various services, including gconfd.

Reply:
Also see the equivalent dbus change for its session bus:

https://github.com/SELinuxProject/refpolicy/pull/671

Reply:
If dbus creates a socket and gconfd_t inherits it, the socket label would not be `gconfd_t`. The fact is that this domain already has a create permission. If you're not going to do the requested change, please close the PR.

Reply:
The create permission comes from: logging_send_syslog_msg

Reply:
Please give me some more time to look at the source code, I have been struggling to find where it eventually creates the socket, so I am still missing something...

Reply:
Unfortunately I could not verify it directly in the source code. However, I have verified it from the syslog files.

The requested change has now been made.